tests/java/IncrementalRebalanceCli.java

```diff
@@ -0,0 +1,69 @@
+import java.io.IOException;
```

Comment:
Copyright blob

Reply:
I'll add this to the other java files too - they don't have it either.